### DISCUSSION TITLE
How to run the bot/image generator without running the code again in collab

### DISCUSSION THREAD

**Discussion by yusofthecodemaster** [lllyasviel/Fooocus#2701] [Q&A] [answered]
and if I download it would it be faster or slower

**Comment by mashb1t** [comment]
please elaborate, what is your goal?

**Reply by yusofthecodemaster** [reply]
when i try to open the program (using the first link) it says no program running so how to use it with the same link motre than once

**Reply by mapluisch** [reply, marked as answer]
If you're talking about the Colab notebook, [Colab notebook instances run up to 12h max](https://research.google.com/colaboratory/faq.html#idle-timeouts)

so you either have to re-run the notebook cells after timing out or just run the project locally - the install instructions are thorough :)
